dags/hivemind_etl_helpers/src/db/github/extract/__init__.py

```diff
@@ -1,5 +1,6 @@
 # flake8: noqa
 from .comments import fetch_comments
 from .commit import fetch_commits
-from .issues import fetch_issues
+from .github_extraction import GithubExtraction
+from .issues import GithubIssueExtraction
```

Comment:
Please consider removing the line 5 (`from .issues import GithubIssueExtraction`) because this is not always required to be used and we're always using GitHubExtraction

If we wanted to use the `GitHubIssueExtraction` we can just import from its file and not this module directory.